features/git-town/configuration/already_set.feature

```diff
@@ -21,7 +21,7 @@ Feature: listing the configuration
         2: production
         3: qa
 
-      Please specify the main development branch by name or number (current value: main): Please specify a perennial branch by name or number. Leave it blank to finish (current value: qa):
+      Please specify the main development branch by name or number (current value: main): Please specify a perennial branch by name or number. Leave it blank to finish (current value(s): qa):
```

Comment:
I think you are more popey than the pope here. The previous version was good enough, everybody understands it, it reads easier, is shorter, and has no nested parentheses.